samples/exemplar/src/main/kotlin/com/squareup/exemplar/dashboard/frontend/FrontendIndexAction.kt

```diff
@@ -0,0 +1,627 @@
+import kotlinx.html.h2
+import kotlinx.html.h3
+import kotlinx.html.header
+import kotlinx.html.id
+import kotlinx.html.img
+import kotlinx.html.input
+import kotlinx.html.label
+import kotlinx.html.li
+import kotlinx.html.nav
+import kotlinx.html.option
+import kotlinx.html.p
+import kotlinx.html.role
+import kotlinx.html.select
+import kotlinx.html.span
+import kotlinx.html.ul
+import kotlinx.html.unsafe
+import misk.config.AppName
+import misk.turbo.buildHtml
+import misk.turbo.turbo_frame
+import misk.web.Get
+import misk.web.PathParam
+import misk.web.ResponseContentType
+import misk.web.actions.WebAction
+import misk.web.dashboard.AdminDashboardAccess
+import misk.web.dashboard.HtmlLayout
+import misk.web.mediatype.MediaTypes
+import wisp.deployment.Deployment
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/**
+ * Example page from Tailwind UI
+ * https://tailwindui.com/components/ecommerce/page-examples/storefront-pages
+ */
+@Singleton
+class FrontendIndexAction @Inject constructor(
+  @AppName private val appName: String,
+  private val deployment: Deployment,
+) : WebAction {
+  @Get("/")
+  @ResponseContentType(MediaTypes.TEXT_HTML)
+  @AdminDashboardAccess
+  fun get(): String {
+    return buildHtml {
+      HtmlLayout(appRoot = "/app", title = "$appName frontend", playCdn = false, appCssPath = "/static/cache/tailwind.exemplar.min.css") {
+        turbo_frame(id = "tab") {
+          div("bg-white") {
+//            +"""<!--
+//    Mobile menu
+//
+//    Off-canvas menu for mobile, show/hide based on off-canvas menu state.
+//  -->"""
+            div("relative z-40 lg:hidden") {
+              role = "dialog"
+              attributes["aria-modal"] = "true"
+//              +"""<!--
+//      Off-canvas menu backdrop, show/hide based on off-canvas menu state.
+//
+//      Entering: "transition-opacity ease-linear duration-300"
+//        From: "opacity-0"
+//        To: "opacity-100"
+//      Leaving: "transition-opacity ease-linear duration-300"
+//        From: "opacity-100"
+//        To: "opacity-0"
+//    -->"""
+              div("fixed inset-0 bg-black bg-opacity-25") {
+              }
+              div("fixed inset-0 z-40 flex") {
+//                +"""<!--
+//        Off-canvas menu, show/hide based on off-canvas menu state.
+//
+//        Entering: "transition ease-in-out duration-300 transform"
+//          From: "-translate-x-full"
+//          To: "translate-x-0"
+//        Leaving: "transition ease-in-out duration-300 transform"
+//          From: "translate-x-0"
+//          To: "-translate-x-full"
+//      -->"""
+                div("relative flex w-full max-w-xs flex-col overflow-y-auto bg-white pb-12 shadow-xl") {
+                  div("flex px-4 pb-2 pt-5") {
+                    button(classes = "-m-2 inline-flex items-center justify-center rounded-md p-2 text-gray-400") {
+                      type = ButtonType.button
+                      span("sr-only") { +"""Close menu""" }
```

Comment:
> +"""Close menu"""

I've been trying to google around to figure out what these string unaryPlus operators are doing. I can tell that it's something to do with kotlinx-html, but the docs are... [not informative](https://github.com/Kotlin/kotlinx.html/wiki/DOM-trees)